src/App.js

```diff
@@ -1,15 +1,19 @@
-import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import Books from './components/pages/books';
-import Categories from './components/pages/categories';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import './styles.css';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
-    <BrowserRouter>
+    <div>
+      <Navbar />
       <Routes>
-        <Route path="/" element={<Books />} />
+        <Route path="/" element={<BookList />} />
         <Route path="/categories" element={<Categories />} />
       </Routes>
```

Comment:
- [ ] Please wrap the following piece of code with `BrowserRouter` for your project to work well, as it is right now, nothing is showing on your website because of that. Please resolve it. :slightly_smiling_face:

Reply:
solved using router v.6 createbrowserroute function